### PR TITLE
Articles schema

### PR DESCRIPTION
**Pull request by pedrohgoncalvess**

### CODE REVIEW

File: migrations/env.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+#url connection
+db_user = os.environ.get('DB_USER')
+db_password = os.environ.get('DB_PASSWORD')
+db_host = os.environ.get('DB_HOST')
+db_name = os.environ.get('DB_NAME')
+
+
+
+db_url = f"mysql+mysqlconnector://{db_user}:{db_password}@{db_host}/{db_name}"
```
Comment (mauricioabreu) [inline migrations/env.py:24]: I think you can reuse the production code here, right? The one that connects to the database and creates an engine, etc

Comment (pedrohgoncalvess) [reply migrations/env.py:23]: I tried to do it using what we have in connection.py but I couldnt. i think that alembic needs to create an engine in every migration. I will read more about it

**Review by josethz00 — APPROVED** [review]
Approved, just add some docs or readme instructions for the migrations later

**Review by mauricioabreu — APPROVED** [review]
Nice!

**Pull request merged**